apps/nextjs-website/src/lib/fetch/types/CTAButton.ts

```diff
@@ -30,3 +32,11 @@ export const CTAButtonSchema = t.intersection([
     size: CTAButtonSize,
   }),
 ]);
+
+export const CTAButton_SimpleSchema = t.strict({
```

Comment:
```suggestion
export const CTAButtonSimpleSchema = t.strict({
```
or even better `CTAButtonSimpleCodec`. At some point we will uniform them form `Schema` to `Codec` or vice versa 😄